defaults/main.yml

```diff
@@ -38,7 +38,7 @@ nginx_extra_root_params: []
 nginx_events_params:
   - worker_connections {% if nginx_max_clients is defined %}{{nginx_max_clients}}{% else %}512{% endif %}
 
-nginx_http_params:
+nginx_http_default_params:
```

Comment:
We can put this variable in the file "vars/main.yml" instead, it should not be changeable by the user